### PR TITLE
adding section 651, 658, 659, 660, 663, 665, 666, 670, 675, 679

### PR DESCRIPTION
**Pull request by Desi7**
### What?
Implemented Kai's section as mentioned in title

### Why?
Because needed

### How?
coding

### Testing?
nope

### Anything Else?
closes: #97
closes: #523 
closes: #527 
closes: #528 
closes: #540 
closes: #542 
closes: #544 
closes: #547 
closes: #550 
closes: #551

### CODE REVIEW

**Review by kaibria — APPROVED** [review]

File: fabbwled-backend/src/main/resources/sections/649-680.yaml [hunk 1]
```diff
@@ -1,4 +1,151 @@
...
+        You untie the little creature from its bonds. It gives a jubilant
+        cry, and soars upwards.
+        ‘Thank you, large one!’ it chatters, ‘My name is Pikalik the
+        Wing-warrior! We never forget – one day I will pay my debt.’
+        With that, he flitters away into the clouds.
+        ‘That’s 75 Shards you just let fly away,’ comments the man
+        in the palanquin wryly, ‘Mannekyn People can’t be trusted,
+        believe me.’
+        You leave the slave market.
+    # TODO: acquireKeyword: Altruist
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/649-680.yaml:48]: this exists

**Review by Nilstrieb — APPROVED** [review]

**Review by Rwael — APPROVED** [review]
Nice😊

**Review by lucasmillanm — APPROVED** [review]
nice

**Review by Lion-alt — APPROVED** [review]
LGTM 👍

**Pull request merged**